code/modules/clothing/head/misc.dm

```diff
@@ -392,3 +392,33 @@
 	desc = "A green Christmas Hat! How festive!"
 	icon_state = "christmashatg"
 	item_state = "christmashatg"
+
+/obj/item/clothing/head/cowboyhat
+	name = "cowboy hat"
+	desc = "A standard brown cowboy hat, yeehaw."
+	icon_state = "cowboyhat"
+	item_state= "cowboyhat"
+
+/obj/item/clothing/head/cowboyhat/black
+	name = "black cowboy hat"
+	desc = "A a black cowboy hat, perfect for any outlaw"
+	icon_state = "cowboyhat_black"
+	item_state= "cowboyhat_black"
+
+/obj/item/clothing/head/cowboyhat/white
+	name = "white cowboy hat"
+	desc = "A white cowboy hat, perfect for your every day rancher"
+	icon_state = "cowboyhat_white"
+	item_state= "cowboyhat_white"
+
+/obj/item/clothing/head/cowboyhat/pink
+	name = "pink cowboy hat"
+	desc = "A pink cowboy? more like cowgirl hat, just don't be a buckle bunny."
+	icon_state = "cowboyhat_pink"
+	item_state= "cowboyhat_pink"
+
+/obj/item/clothing/head/cowboyhat/sec

```

Comment:
Perhaps one might want armor for this...

Reply:
Maybe, might be more suited for a balance PR, have to check if the berets actually have any or not which then I'd go ahead for.